src/components/DayOverview/DayOverview.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import supabase, { Schema } from "@/supabase";
+import DayOverviewButton from "@/components/DayOverview/DayOverviewButton";
+
+export type ParcelOfSpecificDateLocation = {
+    collection_datetime: Schema["parcels"]["collection_datetime"];
+    clients: {
+        flagged_for_attention: Schema["clients"]["flagged_for_attention"];
+        full_name: Schema["clients"]["full_name"];
+        address_postcode: Schema["clients"]["address_postcode"];
+        delivery_instructions: Schema["clients"]["delivery_instructions"];
+    } | null;
+};
+
+export const getParcelsOfSpecificDateLocation = async (
```

Comment:
does this need to be exported?

Reply:
Oh yes, forgot I updated the pages and now it doesn't need to be exported